### ISSUE TITLE
Tab with new unsaved Evernote note should not be closed without dialog query

### ISSUE THREAD

**Issue by rdhyee** [bordaigorl/sublime-evernote#95]
Today I had a new note created with the sublime-evernote plugin that I didn't get around to saving to Evernote. I accidentally hit the Cmd-W button on my Mac to close the tab, losing about an hour work.  What surprised me was that I didn't get any dialog box to prompt me about whether I really did want to close the tab.  Is there some setting I can use to prevent this accidental closing of a tab from happening again?

**Comment by rdhyee** [comment]
It seems that the only way to keep a tab from being accidentally closed is I've used the `Update Evernote Note` function on the note.  Even if I have just used the `Send to Evernote as new note`and made a change to the note, I can close the tab without having to choose between `Close and discard changes` and `Save to Evernote and close`.

**Comment by bordaigorl** [comment]
Can you check you are running the latest version (2.6.0) and the `warn_on_close` setting is set to `true`?

**Comment by rdhyee** [comment]
Yes, I'm running the latest version (2.6.0), but I don't have `warn_on_close` explicitly set.  My understanding that `warn_on_close` defaults to `true`. As I noted [previously](https://github.com/bordaigorl/sublime-evernote/issues/95#issuecomment-105987658).  I do get a warning once I've used `Update Evernote Note` on a note.

**Comment by bordaigorl** [comment]
Ah yes sorry that was a bug which is already corrected in the `devel` branch but has not been released as a version yet.
I might actually publish `2.6.1` with that change alone...to solve your problem until I do that you can clone the repo from the `devel` branch and it should work as expected. I am sorry you lost your note...

**Comment by rdhyee** [comment]
Thanks, @bordaigorl for looking into my problem.  I'm glad the problem is fixed in the `devel` branch.  Yes, it was a pain to loose my note...but I can hardly complain too much given how much I benefit from your fantastic ST-Evernote integration. :-)
